### PR TITLE
railcar: init at 1.0.4

### PR DESCRIPTION
**Pull request by spacekookie**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/virtualization/railcar/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, fetchpatch, rustPlatform, libseccomp }:
+
+rustPlatform.buildRustPackage rec {
+  name = "railcar=${version}";
```
Comment (veprbl) [inline pkgs/applications/virtualization/railcar/default.nix]:
```suggestion
  name = "railcar-${version}";
```

File: pkgs/applications/virtualization/railcar/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    owner = "oracle";
+    repo = "railcar";
+    rev = "v${version}";
+    sha256 = "09zn160qxd7760ii6rs5nhr00qmaz49x1plclscznxh9hinyjyh9";
+  };
+  
+  cargoSha256 = "16f3ys0zzha8l5jdklmrqivl8hmrb9qgqgzcm3jn06v45hls9lan";
+
+  buildInputs = [ libseccomp ];
+  cargoPatches = [ ./cargo-lock.patch ];
```
Comment (rasendubi) [inline pkgs/applications/virtualization/railcar/default.nix]: Would it be a good idea to submit the patch upstream?

Comment (dywedir) [reply pkgs/applications/virtualization/railcar/default.nix]: See https://github.com/oracle/railcar/pull/44

**Comment by rasendubi** [comment]
Added inline comment with the reference to the upstream PR.

@GrahamcOfBorg build railcar

**Pull request merged**